readme.md

```diff
@@ -12,7 +12,13 @@ Administrator is an administrative interface builder for [Laravel](http://larave
 
 ## Composer
 
-To install Administrator as a Composer package to be used with Laravel 4, simply add this to your composer.json:
+To install Administrator as a Composer package to be used with Laravel 5, simply add this to your composer.json:
+
+```json
+"frozennode/administrator": "dev-l5"
```

Comment:
Let's make sure these are up to date with what they would be when merged in. So this should say `dev-master` and the L4 section should say `4.*`.